manager/state/raft/raft.go

```diff
@@ -1092,6 +1151,11 @@ func (n *Node) reportNewAddress(ctx context.Context, id uint64) error {
 	if err != nil {
 		return err
 	}
+	if oldAddr == "" {
```

Comment:
I dunno, there shouldn't be empty addresses and I wonder why it works and registers in peer list at all.